packages/studio-ui/src/web/views/FlowBuilder/diagram/nodes/SkillCallContents/index.tsx

```diff
@@ -21,6 +21,15 @@ type Props = Pick<BlockProps, 'node'>
 const SkillCallContents: FC<Props> = ({ node }) => {
   return (
     <div className={cx(style.contentsWrapper, style['skill-call'])}>
+      {node.previewElements?.map((item, i) => {
```

Comment:
what happens if previewElements is `undefined` ?  react will cry.
If the default value is null then please change the typing to

`previewElements: string[] | null` instead of `previewElements? string[]`

Reply:
I will double check that

Reply:
Not a problem: node.onReceive? today can also be null or undefined